FlowerShop/DTOs/MerchandiseCategoryDTO.cs

```diff
@@ -0,0 +1,34 @@
+﻿using FlowerShop.Models;
+using FlowerShop.Repositories;
+
+namespace FlowerShop.DTOs
+{
+    public class MerchandiseCategoryDTO
```

Comment:
I think MechandiceCategoryDTO should also include child categories so that when we make a request for a category we would also get their child categories.